### ISSUE TITLE
The debugger doesn't show values for bitfields

### ISSUE THREAD

**Issue by moizifty** [EpicGamesExt/raddebugger#189]
Take this code for example:
```
typedef struct s1 s1;
struct s1
{
    int a : 1;
    int b : 1;
};

int main(void)
{
    s1 s = {0};
    s.b = 1;
    
    return 0;
}
```

Inspecting 's' in the debugger shows the following:
![image](https://github.com/EpicGamesExt/raddebugger/assets/45013227/c6534213-273f-4ce7-b1c0-ce92d269572e)

**Comment by ryanfleury** [comment]
What version are you using? This should have been fixed in v0.9.9. If you’re still experiencing it in that version, the first thing to try is rebuild your program. If it still happens after that, then this is a new bug.

**Comment by moizifty** [comment]
> What version are you using? This should have been fixed in v0.9.9. If you’re still experiencing it in that version, the first thing to try is rebuild your program. If it still happens after that, then this is a new bug.

Oof, yea I just tested it with latest, it works as expected!
